tests/test_auth.py

```diff
@@ -46,20 +60,40 @@ def create_app(self):
 
         return self.app
 
+    @classmethod
+    def setUpClass(self):
+        # We only need to create the MockLdap instance once. The content we
+        # pass in will be used for all LDAP connections.
+        self.mockldap = MockLdap(self.directory)
+
+    @classmethod
+    def tearDownClass(self):
+        del self.mockldap
+
     def setUp(self):
         db.create_all()
+        self.mockldap.start()
```

Comment:
It would be nice to keep the ldap tests separate from the other auth tests by sub-classing OrloAuthTest (and moving the pre-existing tests to another sub-class).

While the performance penalty isn't huge, there is no need to start and stop mockldap for the file-based and token tests. Also if ldap functionality is broken, or someone sets up a virtualenv without ldap, all the tests won't work.